perceval/backends/core/mediawiki.py

```diff
@@ -99,7 +99,7 @@ def fetch(self, category=CATEGORY_PAGE, from_date=DEFAULT_DATETIME, reviews_api=
         else:
             from_date = datetime_to_utc(from_date)
 
-        kwargs = {"from_date": from_date, "reviews_api": reviews_api}
+        kwargs = {"from_date": from_date, "reviews_api": reviews_api, "test_mode": self._test_mode}
```

Comment:
`_test_mode` is obsolete. It shouldn't be included in `kwargs` dictionary.
Originally, `_test_mode` was added to bypass a check `MAX_RECENT_DAYS` variable checking while testing this module. 
A better approach would be to mock the call of `datetime.datetime.now(dateutil.tz.tzlocal())` to use always the same date. (See https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/mediawiki.py#L311)